spec/models/user_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  it 'Testing user without name' do
+    user = User.create
+    expect(user).to_not be_valid
+  end
+
+  it 'Testing user PostCounter number negative' do
+    user = User.create(Name: 'John', Image: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'anything')
+    user.PostCounter = -10
+    expect(user).to_not be_valid
+  end
+
+  it 'Name should be present' do
+    user = User.create(Name: 'John', Image: 'https://unsplash.com/photos/F_-0BxGuVvo', Bio: 'anything')
+    user.Name = nil
+    expect(user).to_not be_valid
+  end
+end
```

Comment:
- [ ] As requested by the previous reviewer, Please add a test for the custom method available in the User model `latest_posts`. This test should check 3 recent posts for a user.